include/iarc7_motion/TakeoffController.hpp

```diff
@@ -0,0 +1,89 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Takeoff Controller
+//
+// Handles takeoff
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifndef TAKEOFF_CONTROLLER_H
+#define TAKEOFF_CONTROLLER_H
+
+#include <ros/ros.h>
+
+#include "ros_utils/LinearMsgInterpolator.hpp"
+#include "ros_utils/SafeTransformWrapper.hpp"
+
+// ROS message headers
+#include "iarc7_msgs/OrientationThrottleStamped.h"
+
+namespace Iarc7Motion
+{
+
+enum class TakeoffState { RAMP,
+                          ASCEND,
+                          DONE };
+
+    class TakeoffController
```

Comment:
I meant the other stuff shouldn't be indented, that's what I've been doing and I thought that was our convention